tasks/main.yml

```diff
@@ -118,6 +117,7 @@
        - user.name in admin_logged_in_users
        - user.group is defined
        - user.groups is defined
+       - user.groups == "AnsibleUnsafeText"
```

Comment:
I do not think this condition is formulated correctly. I think it was meant to check the type of the variable, but this is a string comparison. As a result, a logged-in user will not be added to the multiple groups mentioned because this condition is always false (unless the list of group comprises a single group called "AnsibleUnsafeText" of course). Same holds for line 134.

Reply:
Agreed. It looks funny. I'm not in a position to check or test this until September (holiday:) 

https://github.com/CSCfi/ansible-role-users/blob/bd6eb88aee87018096cf64d1d3e6f66e839c4777/tests/test.yml I think should be adjusted to also test this user11 as well.